config.json

```diff
@@ -184,6 +184,13 @@
       ]
     },
     {
+      "slug": "poker",
+      "difficulty": 4,
```

Comment:
Don't know why I thought difficulty was 1-5, but bumped it to 7, just below bowling.